oem/firstvoices/common/build_keyboards.sh

```diff
@@ -58,10 +74,13 @@ while [[ $# -gt 0 ]] ; do
 done
 
 # Clean existing Keyboards folder
+# Can't remove entire folder because Android has other assets/
 
 if [ $DO_CLEAN = true ]; then
   echo "Cleaning target path $KEYBOARDS_TARGET"
-  rm -rf $KEYBOARDS_TARGET/*
+  rm -rf "$KEYBOARDS_TARGET/*.kmp"
+  rm -rf "$KEYBOARDS_TARGET/*.js"
```

Comment:
Let's not remove .js now -- only the .kmp and .keyboard_info -- the iOS code will shortly be consuming the .kmp anyway. There's risk in removing *.js because assets folder may contain .js in the future.